build_tools/python/e2e_test_framework/models/tf_models.py

```diff
@@ -8,36 +8,38 @@
 from e2e_test_framework import unique_ids
 from e2e_test_framework.definitions import common_definitions
 
+TF_MODELS_MANUAL_ROOT_DIR = "https://storage.googleapis.com/iree-model-artifacts/tensorflow/manual"
+
 MINILM_L12_H384_UNCASED_INT32_SEQLEN128 = common_definitions.Model(
     id=unique_ids.MODEL_MINILM_L12_H384_UNCASED_INT32_SEQLEN128,
     name="MiniLML12H384Uncased",
     tags=["int32", "seqlen128"],
-    source_type=common_definitions.ModelSourceType.EXPORTED_TF_V2,
+    source_type=common_definitions.ModelSourceType.EXPORTED_MHLO_MLIR,
     # Converted from https://huggingface.co/microsoft/MiniLM-L12-H384-uncased/commit/44acabbec0ef496f6dbc93adadea57f376b7c0ec
     source_url=
-    "https://storage.googleapis.com/iree-model-artifacts/minilm-l12-h384-uncased-seqlen128-tf-model.tar.gz",
+    f"{TF_MODELS_MANUAL_ROOT_DIR}/MiniLML12H384Uncased_2023-05-07.timestamp_1683504734.mlirbc",
```

Comment:
I found that we are not uploading `.mlirbc` to the GCS. We need to update:

https://github.com/openxla/iree/blob/3e942c672688841fac553cb4e4d53e47b22ca9fb/.github/workflows/build_e2e_test_artifacts.yml#L126

to be `-name "iree_*" -o -name "model_*.mlir" -o -name "model_*.mlirbc" -o -name "*.json" | \`

Reply:
Updated